doc/source/Getting_started/Installation.rst

```diff
@@ -44,14 +44,14 @@ Starting from 2023R2, buttons are available in the Automation Tab as in the exam
   :alt: PyAEDT toolkit buttons available in AEDT
 
 
-Toolkit Manager
-~~~~~~~~~~~~~~~
-The user can install or uninstall automated workflows using the toolkit manager.
+Extension manager
+~~~~~~~~~~~~~~~~~
+The user can install or uninstall automated workflows using the extension manager.
 There are three options:
 
 - Custom PyAEDT scripts.
 
-- Internal PyAEDT scripts. Existing workflows in the PyAEDT library.
+- PyAEDT workflows. Existing workflows in the PyAEDT library.
 
 - Open source PyAEDT toolkits. Open source PyAEDT toolkits following the `common library <https://aedt.common.toolkit.docs.pyansys.com/>`_ standard.
```

Comment:
```suggestion
- Open source PyAEDT toolkits described in the `PyAEDT Common Toolkit <https://aedt.common.toolkit.docs.pyansys.com/>`_ documentation.
```
The title for the referenced doc should be "PyAEDT Comon Toolkit" documentation followed by the version. I opened a PR to fix it.